lib/cli-create-token.ts

```diff
@@ -56,7 +56,7 @@ export default function CliCreateToken() {
                     token = powerbi.PowerBIToken.createDevToken(settings.collection, settings.workspace);
                     break;
                 case 'provision':
-                    if (!(settings.collection && settings.workspace)) {
+                    if (!(settings.collection)) {
```

Comment:
Since there is only 1 parameter now we could remove the extra set of parentheses: `!settings.collection` to make it slightly cleaner although it doesn't matter much.